src/Chapter2.hs

```diff
@@ -620,7 +648,8 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [a] -> [a]
+takeEven xs = fst <$> (filter (\(_,b) -> even b) $ zip xs [0..])
```

Comment:
Very creative! 🎨 

Note that you can implement this function with the simple pattern matching as well. It requires fewer function calls and maybe (just maybe) a bit more efficient (at least by memory).